sympy/integrals/transforms.py

```diff
@@ -1860,7 +1870,10 @@ def laplace_transform(f, t, s, legacy_matrix=True, **hints):
 
     The implementation is rule-based, and if you are interested in which
     rules are applied, and whether integration is attempted, you can switch
-    debug information on by setting ``sympy.SYMPY_DEBUG=True``.
+    debug information on by setting ``sympy.SYMPY_DEBUG=True``. The numbers
+    of the rules in the debug information (and the code) refer to Bateman's
+    Tables of Integral Transforms Vol. 1, McGraw-Hill 1954, available here:
+    https://resolver.caltech.edu/CaltechAUTHORS:20140123-101456353
```

Comment:
We can use a sphinx reference for this:
https://docs.sympy.org/latest/contributing/docstring.html#references-section

Reply:
Change (Q)